src/odb/README.md

```diff
@@ -89,15 +78,327 @@ database to have exactly the same layout across save/restores.
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-domain_name` | TBC. |
+| `-area` | TBC. |
+
+### Delete Physical Cluster
+
+Description TBC.
+
+```tcl
+delete_physical_cluster cluster_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `cluster_name` | TBC. |
+
+### Delete Voltage Domain
+
+Description TBC.
+
+```tcl
+delete_voltage_domain domain_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `domain_name` | TBC. |
+
+### Assign Power Net
+
+Description TBC.
+
+```tcl
+assign_power_net 
+    -domain domain_name
+    -net snet_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-domain_name` | TBC. |
+| `-net` | TBC. |
+
+### Assign Ground Net
+
+Description TBC.
+
+```tcl
+assign_ground_net
+    -domain domain_name
+    -net snet_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-domain_name` | TBC. |
+| `-net` | TBC. |
+
+### Add to Physical Cluster
+
+Description TBC.
+
+```tcl
+add_to_physical_cluster
+    [-modinst path]
+    cluster_name
+or 
+add_to_physical_cluster
+    [-inst inst_name]
+    cluster_name
+or
+add_to_physical_cluster
+    [-physical_cluster cluster_name]
+    cluster_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-modinst` | TBC. |
+| `-inst` | TBC. |
+| `-physical_cluster` | TBC. |
+| `cluster_name` | TBC. |
+
+### Remove From Physical Cluster
+
+Description TBC.
+
+```tcl
+remove_from_physical_cluster
+    [-parent_module module_name]
+    [-modinst modinst_name]
+    cluster_name
+or
+remove_from_physical_cluster
+    [-inst inst_name]
+    cluster_name
+or
+remove_from_physical_cluster
+    [-physical_cluster cluster_name]
+    cluster_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-parent_module` | TBC. |
+| `-modinst` | TBC. |
+| `-inst` | TBC. |
+| `-physical_cluster` | TBC. |
+| `-cluster_name` | TBC. |
+
+### Report Physical Clusters
+
+Description TBC.
+
+```tcl
+report_physical_clusters
+```
+
+### Report Voltage Domains
+
+Description TBC.
+
+```tcl
+report_voltage_domains
+```
+
+### Report Group
+
+Description TBC.
+
+```tcl
+report_group group
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `group` | TBC. |
+
+### Write Guides
+
+This command writes global routing guides, which can be used as input 
+for global routing.
+
+Example: `write_guides route.guide`.
+
+```tcl
+write_guides file_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `file_name` | Guide file name. |
+
+### Write Macro Placement
+
+This command writes macro placement.
+
+```tcl
+write_macro_placement file_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `file_name` | Macro placement file name. |
+
+
+
+
 ## Example scripts
 
+After building successfully, run OpenDB Tcl shell using
+`../../build/src/odb/src/swig/tcl/odbtcl`. An example usage:
```

Comment:
We dropped user support for odbtcl quite a while ago.  Such commands should just be run in OR.   The binary is still there just for testing; it is no longer installed outside of the build/ hierarchy.

Reply:
Do you recommend we remove this line?

Reply:
I would remove any mention of odbtcl